components/HomeScreen/Title/Title.datalayer.jsx

```diff
@@ -0,0 +1,20 @@
+import { useState } from "react";
+import Title from "./Title";
+import { yieldRotateGlobe } from "./Title.utils";
+
+
+function TitleDatalayer() {
+
+  const [globeOrientation, setGlobeOrientation] = useState(1);
+
+  const onPress = yieldRotateGlobe(globeOrientation, setGlobeOrientation);
```

Comment:
code golf - no real reason to create the onPress here, could just call the function directly on line 15.